src/main/java/com/exadel/discount/repository/CategoryRepository.java

```diff
@@ -10,6 +10,9 @@
 
 @Repository
 public interface CategoryRepository extends JpaRepository<Category, UUID> {
+
     @EntityGraph(attributePaths = {"discounts"})
     Optional<Category> findById(UUID uuid);
+
+    Optional<Category> findCategoryById(UUID id);
```

Comment:
What is the difference between findById and findCategoryById methods?

Reply:
findById load entity category with all its discounts(its need for sorting)
findCategoryById load only entity category
I don't need all discounts, to will be edit category